Loop/View Models/SimpleBolusViewModel.swift

```diff
@@ -99,6 +104,22 @@ class SimpleBolusViewModel: ObservableObject {
             _manualGlucoseString = newValue
         }
     }
+    
+    private func updateNotice() {
+        let minRecommendationGlucose = displayMealEntry ? LoopConstants.simpleBolusCalculatorMinGlucoseMealBolusRecommendation : LoopConstants.simpleBolusCalculatorMinGlucoseBolusRecommendation
+        
+        switch manualGlucoseQuantity {
+        case let g? where g < minRecommendationGlucose:
+            activeNotice = .glucoseBelowRecommendationLimit
+        case let g? where g < LoopConstants.simpleBolusCalculatorGlucoseWarningLimit:
```

Comment:
[nit] This case is never going to happen with the existing constants. Is this intended?

Reply:
During meal bolus, `minRecommendationGlucose` will be 55, and `simpleBolusCalculatorGlucoseWarningLimit` is 70, so this case will be used if the entered glucose is between those.